volttron/platform/vip/agent/subsystems/ping.py

```diff
@@ -82,6 +82,8 @@ def ping(self, peer, *args):
         result = next(self._results)
         args = list(args)
         args.insert(0, b'ping')
+        if peer == b'pubsub':
+            peer = b''
```

Comment:
If at all possible, we need to preserve backwards compatibility on such a central piece of the  platform.